mr/src/test/java/org/apache/iceberg/mr/hive/HiveIcebergStorageHandlerBaseTest.java

```diff
@@ -157,6 +177,282 @@ public void testJoinTables() throws IOException {
     Assert.assertArrayEquals(new Object[] {1L, "Bob", 102L, 33.33d}, rows.get(2));
   }
 
+  @Test
+  public void testCreateDropTable() throws TException, IOException {
+    // We need the location for HadoopTable based tests only
+    String location = locationForCreateTable(temp.getRoot().getPath(), "customers");
+    shell.executeStatement("CREATE EXTERNAL TABLE customers " +
+        "STORED BY 'org.apache.iceberg.mr.hive.HiveIcebergStorageHandler' " +
```

Comment:
Just thinking about how this interacts with #1505: This PR relies on setting the storage handler here and it is stored in parameters. Right now, nothing will remove it, but I think the right thing to do in #1505 is to remove this if Hive is not enabled for the table.

If we remove the property when Hive isn't enabled, then this test would break. To fix it, I think we should always set the `engine.hive.enabled` property to true in the hook. That could be done either in this PR or the other one if this one is merged first.

Reply:
Done